pkg/autoscaler/autoscaler.go

```diff
@@ -51,8 +51,14 @@ type Stat struct {
 	// Average number of requests currently being handled by this pod.
 	AverageConcurrentRequests float64
 
+	// Similar to AverageConcurrentRequests, but for requests going through a proxy.
```

Comment:
Judging by the below (`fallthrough` in switch), should we state in a comment somewhere that `AverageConcurrentRequests` is calculated over **all** requests and that `AverageProxiedConcurrency` is a subset of that, for requests that are proxied?

Also: Do we want to adjust the naming for both to be identical, i.e. rename the new value to `AverageProxiedConcurrentRequests`?

Reply:
Done.  Although I think these names should be shorter, to follow go convention of short names.